src/libstd/collections/hash/set.rs

```diff
@@ -1198,7 +1198,7 @@ mod test_set {
 
     #[test]
     fn test_drain() {
-        let mut s: HashSet<int> = (1is..100).collect();
+        let mut s: HashSet<i32> = (1..100).collect();
```

Comment:
Not critical at all, but this explicit `i32` could probably be left out in cases like this and instead just `_` could be used.